### PR TITLE
Feat: Added Typography and Text utilities

### PR DESCRIPTION
**Pull request by coldpigli**
Added some text utilities and typography

### CODE REVIEW

File: docs/typography-page.html [hunk 1]
```diff
@@ -128,39 +139,184 @@ <h4 class="sub-title gap-d20">COMPONENTS</h4>
             </div>
         </section>
     </aside>
+
+    <!-- Main Content -->
     <main class="content">
         <h1 class="txt-xlg txt-bold gap-d20">
```
Comment (vaibhav-mahalle) [inline docs/typography-page.html:145]: what is d20

**Review by vaibhav-mahalle — COMMENTED** [review]
everything else looks good.

**Review by krishanu2911 — COMMENTED** [review]
Everything is good to go no changes are required according to me

File: docs/typography-page.html (hunk 1 above)
Comment (coldpigli) [reply docs/typography-page.html:145]: d-20 is a utility class i have that it stands for gap-down-20pixels

**Pull request merged**